packages/benchmark-test/tools/benchmark.js

```diff
@@ -0,0 +1,39 @@
+// import { performance } from 'perf_hooks';
+const performance = { now: () => Date.now() };
```

Comment:
I did not change anything in this file beyond moving it to "tools/". However, I considered changing this line to

```suggestion
// import { performance } from 'perf_hooks';
const performance = { now: () => Date.now() * 1000 };
```
Since your naming it "performance.now" suggests that we're using it as a low resolution replacement on those platforms on which it is absent. I did not make this change because I'm not sure what else I would then need to change.

Reply:
I am of this opinion that we should start writing tests with `Date.now` by merging the PR. Meanwhile I can work on something that gives us time in nano seconds.

Reply:
Ok. But should we multiply the output of `Date.now()` by `1000` to better prepare for switching to the real `performance.now()` eventually, on platforms where that is available?

Reply:
For now, I am keeping it consistent with both engines. This means that even if `node` has the `perf_hooks` I am still gonna use `Date.now` for both.
But, I am inclined to see the time in `nano seconds` because this is where we are heading. So, it will be better to not give developers any surprise. One day they wake up and update the package and see all of their tests are failing because they used the `ms` instead of `ns`.

Reply:
Since the benchmarking is in our control, why not multiply by `1000` regardless wether we use `Date.now` or `performance.now`?
If we keep it in nanoseconds, we can benefit later when we actually have nanoseconds precision without breaking anything